src/sentry/static/sentry/app/stores/alertStore.jsx

```diff
@@ -9,34 +10,43 @@ const AlertStore = Reflux.createStore({
     this.count = 0;
   },
 
-  onAddAlert(message, type, expireAfter, url) {
-    // intentionally recreate array via concat because of Reflux
-    // "bug" where React components are given same reference to tracked
-    // data objects, and don't *see* that values have changed
-    let alertId = this.count++;
-
-    this.alerts = this.alerts.concat([{
-      id: alertId,
-      message: message,
-      type: type,
-      url: url
-    }]);
+  getMutedKey(alert) {
+    return `alerts:${alert.id}:muted`;
+  },
 
-    if (typeof expireAfter === 'undefined') {
-      expireAfter = 5000;
+  onAddAlert(alert) {
+    if (alert.id !== void 0) {
```

Comment:
I don't know which is more idiomatic (or house style) — comparing to `void 0` or `typeof == "undefined"` or if it even really matters.